api-server/src/roster/models.py

```diff
@@ -481,13 +481,27 @@ class LiverySkin(models.Model):
         ]
     }
     name = models.CharField(max_length=128)
-    dds_file = models.FileField(upload_to='livery_dds', unique=True)
+    dds_file = models.FileField(upload_to='livery_dds')
     json_description = models.JSONField(
         blank=True,
         null=True,
         validators=[LiverySkinJsonValidator(limit_value=JSON_FIELD_SCHEMA)]
     )
 
+    # Allows for overwrites of the existing dds_file with the same name
```

Comment:
we might be be able to use the update method on the model serializer instead of rolling our own here

Reply:
looking into this

Reply:
https://www.django-rest-framework.org/api-guide/generic-views/#retrieveupdatedestroyapiview